andela/anbu-cfh
# Feature/134249283/users should be able to create start a new game

#### What does this PR do?
Allows users to create and start new game
#### Description of Task to be completed?
Users should be able to start a new game session after login or signup. 
Custom game sessions should be saved to a database
api end points for post and put request for games should be created
#### How should this be manually tested?
sign into the game, click on play with friends, a dialog box will appear asking you to proceed, click on proceed.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#134249283
#### Screenshots (if appropriate)
N/A
#### Questions:
